### PR TITLE
Apache StormCrawler 3.0 (Incubating)

### PR DESCRIPTION
**Pull request by rzo1**
https://lists.apache.org/thread/88nkdyb3dcblxof16pszom9tqtsdhf3x

### CODE REVIEW

File: external/opensearch/archetype/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ under the License.
   <parent>
     <groupId>org.apache.stormcrawler</groupId>
     <artifactId>stormcrawler</artifactId>
-    <version>3.0-SNAPSHOT</version>
+    <version>3.1-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (rzo1) [reply external/opensearch/archetype/pom.xml:28]: I will change this via a separate PR.

[the remark this answers is not in the document]
Comment (rzo1) [reply external/opensearch/archetype/pom.xml:28]: Ah let my quickly update, so no need to double review it again.

[the remark this answers is not in the document]
Comment (rzo1) [reply external/opensearch/archetype/pom.xml]: Fixed.

**Review by ayushtkn — APPROVED** [review]

**Comment by rzo1** [comment]
Thanks for your reviews. I will put that on main and send the announce soon!

**Pull request merged**